### PR TITLE
netdata/packaging: Expect .tar.gz version of go.d plugin

### PR DESCRIPTION
**Pull request by paulkatsoulakis**
##### Summary
Expect .tar.gz version of go.d plugin, after go.d …PR248

MUST BE MERGED AFTER GO.D deployment with the tarball https://github.com/netdata/go.d.plugin/pull/248
##### Component Name
netdata/packaging

##### Additional Information
None

### CODE REVIEW

**Comment by ilyam8** [comment]
> MUST BE MERGED AFTER GO.D deployment with the tarball netdata/go.d.plugin#248

Since https://github.com/netdata/go.d.plugin/pull/246 is merged do we need this PR and https://github.com/netdata/netdata/pull/6557 are connected as separate PRs?

Both of them changing go.d.plugin installer.

**Comment by ilyam8** [comment]
> Since netdata/go.d.plugin#246 is merged do we need this PR and #6557 as separate PRs?

We need in one PR:
 - expect `.tar.gz` version of `go.d.plugin`
 - update version to `0.8.0` (version, checksums)
 - put `go.d.plugin` version in one place [optional]

**Comment by ilyam8** [comment]
clean install

```bash
 --- Install go.d.plugin --- 
[/home/ilyam/src/netdata_test]# curl -sSL --connect-timeout 10 --retry 3 https://github.com/netdata/go.d.plugin/releases/download/v0.8.0/go.d.plugin-v0.8.0.linux-amd64.tar.gz 
 OK   

[/home/ilyam/src/netdata_test]# curl -sSL --connect-timeout 10 --retry 3 https://github.com/netdata/go.d.plugin/releases/download/v0.8.0/config.tar.gz 
 OK   

go.d.plugin-v0.8.0.linux-amd64.tar.gz: OK
config.tar.gz: OK
[/home/ilyam/src/netdata_test]# rm -rf /opt/netdata/usr/lib/netdata/conf.d/go.d 
 OK   

[/home/ilyam/src/netdata_test]# rm -rf /opt/netdata/usr/lib/netdata/conf.d/go.d.conf 
 OK   

[/home/ilyam/src/netdata_test]# tar -xf /tmp/netdata-go-bxEYAR/config.tar.gz -C /opt/netdata/usr/lib/netdata/conf.d/ 
 OK   

[/home/ilyam/src/netdata_test]# chown -R root:netdata /opt/netdata/usr/lib/netdata/conf.d 
 OK   

[/home/ilyam/src/netdata_test]# tar xf go.d.plugin-v0.8.0.linux-amd64.tar.gz -C /opt/netdata/usr/libexec/netdata/plugins.d 
tar: go.d.plugin-v0.8.0.linux-amd64.tar.gz: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
 FAILED   

[/home/ilyam/src/netdata_test]# chown root:netdata /opt/netdata/usr/libexec/netdata/plugins.d/go.d.plugin 
chown: cannot access '/opt/netdata/usr/libexec/netdata/plugins.d/go.d.plugin': No such file or directory
 FAILED   

[/home/ilyam/src/netdata_test]# chmod 0750 /opt/netdata/usr/libexec/netdata/plugins.d/go.d.plugin 
chmod: cannot access '/opt/netdata/usr/libexec/netdata/plugins.d/go.d.plugin': No such file or directory
 FAILED 
```

**Comment by paulkatsoulakis** [comment]
there was a slash missing, fixed, thanks Ilya!

File: netdata-installer.sh [hunk 1]
```diff
@@ -833,7 +833,7 @@ install_go() {
 		run tar -xf "${tmp}/config.tar.gz" -C "${NETDATA_STOCK_CONFIG_DIR}/"
 		run chown -R "${ROOT_USER}:${NETDATA_GROUP}" "${NETDATA_STOCK_CONFIG_DIR}"
 
-		run tar xf "${GO_PACKAGE_BASENAME}" -C "${NETDATA_PREFIX}/usr/libexec/netdata/plugins.d"
+		run tar xf "${GO_PACKAGE_BASENAME}" -C "${NETDATA_PREFIX}/usr/libexec/netdata/plugins.d/"
```
Comment (ilyam8) [inline netdata-installer.sh]: > run tar xf "${GO_PACKAGE_BASENAME}"

> run tar xf "${tmp}/${GO_PACKAGE_BASENAME}"

Comment (paulkatsoulakis) [reply netdata-installer.sh]: hah! line 816 distracted me, which is totally different usage

**Comment by ilyam8** [comment]
@paulkatsoulakis please update checksums

File: contrib/debian/install_go.sh [hunk 2]
```diff
@@ -88,7 +88,10 @@ install_go() {
 
 		# Install files
 		tar -xf "${tmp}/config.tar.gz" -C "${LIB_DIR}/conf.d/"
-		mv "${tmp}/$GO_PACKAGE_BASENAME" "${LIBEXEC_DIR}/plugins.d/go.d.plugin"
+		tar xf "${tmp}/${GO_PACKAGE_BASENAME}"
```
Comment (ilyam8) [inline contrib/debian/install_go.sh]: > tar xf "${tmp}/${GO_PACKAGE_BASENAME}"

-C ${tmp}

Comment (paulkatsoulakis) [reply contrib/debian/install_go.sh]: no need

Comment (ilyam8) [reply contrib/debian/install_go.sh]: i wrote it before `nit` commit actually

**Pull request merged**